code/__HELPERS/mobs.dm

```diff
@@ -52,12 +52,18 @@
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/human, GLOB.tails_list_human)
 	if(!GLOB.tails_list_lizard.len)
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/lizard, GLOB.tails_list_lizard)
+	if(!GLOB.tails_list_ethari.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/ethari, GLOB.tails_list_ethari)
 	if(!GLOB.snouts_list.len)
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/snouts, GLOB.snouts_list)
+	if(!GLOB.snouts_ethari_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/snouts_ethari, GLOB.snouts_ethari_list)
+	if(!GLOB.ears_ethari_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/anthro_ears, GLOB.ears_ethari_list)
 	if(!GLOB.horns_list.len)
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/horns, GLOB.horns_list)
 	if(!GLOB.ears_list.len)
-		init_sprite_accessory_subtypes(/datum/sprite_accessory/ears, GLOB.horns_list)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/ears, GLOB.ears_list)

```

Comment:
This was someone else's error; irrelevant to the PR but I fixed it while I was there.